DANS-KNAW/easy-fedora2vault
# EASY-2683 : convert most DDM elements to EMD elements

Fixes EASY-2683 emd2ddm

#### When applied it will...
* convert most EMD elements to DDM elements
* 
* 

#### Where should the reviewer @DANS-KNAW/easy start?

This DDM(emd.xml) looks like DDM(json) of easy-deposit-api: the input is different.

#### How should this be manually tested?

as #3 or build, deploy, on deasy:
```
easy-fedora2vault -d easy-dataset:17 -o bags simple
easy-validate-dans-bag --aip --bag-store . bags/UUID
```

Note that the interview of `easy-dataset:10` takes quite a while to copy.


Currently validation complains about:
* `metadata/license.pdf` that should not exist
* missing`metadata/files.xml` for old datasets, invalid content for a dataset from the new deposit form

#### Related pull requests on github

repo                       | PR                | note
-------------------------- | ----------------- | ----
easy-                      | [PR#](PRlink)     | ..
